sources/packages/api/src/auth/roles.enum.ts

```diff
@@ -2,5 +2,34 @@
  * Application roles to be used in alongside with RolesGuard.
  */
 export enum Role {
+  /**
+   * Student roles.
```

Comment:
Can we check if it is used, I believe that it can be removed.

Reply:
actually, I was about to remove this. then I realized that its use in 
![image](https://user-images.githubusercontent.com/77353155/183724965-c1a9dcc5-98ce-4ff8-bb7e-45ee4202d252.png)

Reply:
its a test controller, i think it can be removed